src/MovieDetails/MovieDetails.js

```diff
@@ -1,10 +1,19 @@
 import React from 'react';
 import './MovieDetails.css';
 
-const MovieDetails = () => {
+const MovieDetails = ({ movie }) => {
     return (
-        <p>Party!</p>
+        <section>
```

Comment:
Great! Now we have full details displaying. We can add a className to section for stylings. Looks like it may be inheriting the centered content from header?